yasa/hypno.py

```diff
@@ -83,6 +83,15 @@ class Hypnogram:
     >>> from yasa import Hypnogram
     >>> values = ["W", "W", "W", "S", "S", "S", "S", "S", "W", "S", "S", "S"]
     >>> hyp = Hypnogram(values, n_stages=2)
+    >>> hyp
+    <Hypnogram | 12 epochs x 30s (6.0 minutes), 2 stages>
+     - Use `.hypno` to get the values as a pandas.Series
+     - Use `.as_int()` to get the values as integers
```

Comment:
Maybe detail to help differentiate, something like:

- `Use .hypno to get the string values as a pandas.Series`
- `Use .as_int() to get the integer values as a pandas.Series`